workflow/rocoto/gfs_tasks.py

```diff
@@ -554,16 +554,36 @@ def _fcst_cycled(self):
     def post(self):
         add_anl_to_post = False
         if self.app_config.mode in ['cycled']:
-            add_anl_to_post = True
+            add_anl_to_post = False
 
         return self._post_task('post', add_anl_to_post=add_anl_to_post)
 
+    def postanl(self):
+        postenvars = self.envars.copy()
+        postenvar_dict = {'FHRGRP': 'anl',
+                          'FHRLST': 'anl',
+                          'ROTDIR': self._base.get('ROTDIR')}
```

Comment:
`FHRGRP` and `FHRLST` should not be needed anymore since they are constant. It can just be set directly in `jobs/rocoto/postanl.sh`.

Reply:
@HenryWinterbottom-NOAA This one

Reply:
@WalterKolczynski-NOAA `FHRLST` is required.